build/bin/kylin.sh

```diff
@@ -143,9 +143,9 @@ function retrieveDependency() {
 
     # compose hadoop_dependencies
     hadoop_dependencies=${hadoop_dependencies}:`hadoop classpath`
-#    if [ -n "${hbase_dependency}" ]; then
-#        hadoop_dependencies=${hadoop_dependencies}:${hbase_dependency}
-#    fi
+    if [ -n "${hbase_dependency}" ]; then
```

Comment:
From my side, I would like to remove HBase related jars in the future. So, I find it will break the this CubeMigrationTool .

Reply:
Agreed ! It breaks the long term goal.